### PR TITLE
Split checkout docs

### PR DESCRIPTION
**Pull request by timuric**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -13,7 +13,18 @@ module.exports = {
...
-        "developer/checkout",
+        {
+          type: "category",
+          label: "Checkout",
+          items: [
+            "developer/checkout/overview",
+            "developer/checkout/lines",
+            "developer/checkout/stock",
+            "developer/checkout/address",
+            "developer/checkout/finilizing",
```
Comment (mmiszy) [inline sidebars.js:24]:
```suggestion
            "developer/checkout/finalizing",
```

File: docs/developer/checkout/finilizing.mdx
Comment (mmiszy) [inline docs/developer/checkout/finilizing.mdx:1]: typo: `finalizing`

**Review by mmiszy — CHANGES_REQUESTED** [review]

**Review by korycins — APPROVED** [review]

**Review by 2can — APPROVED** [review]

**Comment by 2can** [comment]
- I will remove the "Checkout" tile from the home screen in the separate PR.
- regarding the "Core concepts" it's a matter of naming, both seems fine to me

**Review by zaiste — APPROVED** [review]

**Review by mmiszy — APPROVED** [review]

File: sidebars.js [hunk 2]
```diff
@@ -13,7 +13,18 @@ module.exports = {
...
+          label: "Checkout",
+          items: [
+            "developer/checkout/overview",
+            "developer/checkout/lines",
+            "developer/checkout/stock",
+            "developer/checkout/address",
+            "developer/checkout/finalizing",
+            "developer/checkout/order-to-checkout",
+          ],
+        },
```
Comment (patrys) [inline sidebars.js:27]: You can auto-generate sections based on file structure

**Pull request merged**